### ISSUE TITLE
CreateEvent - CurrDate

### ISSUE THREAD

**Issue by gen1e** [Women-in-Computing-at-RIT/AlumniApp#4]
Need to add a function that will get the current date.

**Comment by glossawy** [comment]
Does that make sense? React components are _stateless_, they are functional constructs that should remain pure functional as they were designed with one-way dataflow. You put data in via properties but you don't get data back. The exception being actions/callbacks passed to elements by their parents ("properties flow down, actions flow up"). 

*Storing state in a react component is generally an anti-pattern* and you ultimately have to fight React to do anything meaningful.

Maintaining and acting on state is the role Flux and Redux (improved Flux) are intended to fill. They essentially make your app a giant state machine and keep the React components pure by only allowing you to map state to react element properties. 

Edit: I should note there is of course some ability to do this. React components *do* have state objects attached to them. But this is usually starts to become unwieldy when attempting to communicate across systems or between systems that don't share the same parent (you have to use actions to tell the parent to communicate with another child or to communicate further up the chain. Which is hard to follow.) There is a page on stateful react components, but the more stateless a react component is, the better, typically.
